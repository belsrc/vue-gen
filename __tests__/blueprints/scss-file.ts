/* eslint-disable fp-jxl/no-unused-expression, fp-jxl/no-nil */
import genScss from '../../src/blueprints/scss-file';

const className = 'test-component';

describe('genScss', () => {
  test('has the expected ouput structure', () => {
    const actual = genScss(className);

    expect(actual).toMatchSnapshot();
  });
});
