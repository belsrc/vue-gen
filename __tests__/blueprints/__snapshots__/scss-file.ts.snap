// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`genScss has the expected ouput structure 1`] = `
"// Mobile / Baseline
.test-component {}

// Tablets
@media only screen and (min-width: 768px) {}

// Hi-Res Tablets
@media only screen and (min-width: 1024px) {}

// Desktops
@media only screen and (min-width: 1200px) {}

// Larger Desktops
@media only screen and (min-width: 1920px) {}
"
`;
